src/main/resources/application.yml

```diff
@@ -33,8 +33,8 @@ management:
 
 store:
   cardano:
-    host: ${REMOTE_NODE_URL:relays-new.cardano-mainnet.iohk.io}
-    port: ${REMOTE_NODE_PORT:3001}
+    host: ${REMOTE_NODE_URL:192.168.68.113}
```

Comment:
we may not want to check this in

Reply:
dang, this was an unwanted change, I was testing against my local node 🤦